### DISCUSSION TITLE
MiniCssExtractPlugin for gulp?

### DISCUSSION THREAD

**Discussion by jamacoe** [webpack/webpack#17198] [Q&A]
I'm using gulp. Is there any port or fork of MiniCssExtractPlugin to work with gulp? Or any advice which npm plugin works similar? I've discovered [https://www.npmjs.com/package/mini-css-extract-plugin](https://www.npmjs.com/package/mini-css-extract-plugin) but it states that it needs Webpack 5 up and running.

**Comment by alexander-akait** [comment]
You can use old version of mini-css-extract-plugin which support webpack v4

**Reply by jamacoe** [reply]
So I can use such an old version with gulp and without webpack? Or would I need webpack anyway? Which 'old' version do you exactly mean?

**Reply by alexander-akait** [reply]
You need webpack, this plugin doesn't work without webpack
